rbac/api/cross_access/serializer.py

```diff
@@ -95,3 +95,16 @@ def create(self, validated_data):
         for role in roles:
             request.roles.add(role)
         return request
+
+    def update(self, instance, validated_data):
+        """Override the update method to associate the roles to cross account request after it is updated."""
+        role_data = validated_data.pop("roles")
+        display_names = [role["display_name"] for role in role_data]
+        for field in validated_data:
+            setattr(instance, field, validated_data.get(field))
+
+        roles = Role.objects.filter(display_name__in=display_names)
```

Comment:
Based on dev sync, we'll likely want to add a filter on `system` here similar to: https://github.com/RedHatInsights/insights-rbac/pull/424/files#diff-9b7f444eec2c52f6eb5578161c2077e0aa9037fbaa6c8ed89eefdcc46dfee381R192